Project.toml

```
@@ -38,6 +39,7 @@ TenetChainRulesTestUtilsExt = ["ChainRulesCore", "ChainRulesTestUtils"]
 TenetDaggerExt = "Dagger"
 TenetFiniteDifferencesExt = "FiniteDifferences"
 TenetGraphMakieExt = ["GraphMakie", "Makie"]
+TenetKrylovKitExt = ["KrylovKit"]
```

```suggestion
TenetKrylovKitExt = "KrylovKit"
```
We can remove this brackets, right? Other from this, this PR is ready to merge for me.